src/Tgstation.Server.Host.Console/tgs.service

```
@@ -0,0 +1,10 @@
+[Unit]
+Description=tgstation-server
+After=network.target
```

```suggestion
After=network.target
After=mariadb.target
```
99% of linux installs will be using MariaDB as the database backing, and you will want TGS to launch after that.